### ISSUE TITLE
Surface More_Info tag in RockOn library before installation

### ISSUE THREAD

**Issue by Hooverdan96** [rockstor/rockstor-core#2231]
Based on conversations with @FroggyFlox (see PR https://github.com/rockstor/rockon-registry/pull/238), I am suggesting that the More_Info tag is surfaced in the RockOn library before installation. Some of the information that would be relevant is not available until after the more_info tag shows up on the installed RockOn. If that icon could be available in the uninstalled library already, then it would not clutter the base information, but provide additional consideration (where necessary) in the same pop-up style.

**Comment by Hooverdan96** [comment]
May be this could be realized by designing the UI with a collapsible section (collapsed by default) per Rockon to avoid the visual clutter, but still allow for pertinent info prior to installation being available to the user

**Comment by phillxnet** [comment]
@Hooverdan96 That would be nice. However we could similarly just add the existing 'i' icon to the uninstalled display of a Rock-on: at least for now. I'm assuming here that we have already 'injested' this info within the DB. If so then we may just be able to duplicate our 'i' icon front end code over to the uninstalled view of a Rock-on. That would at least make this info available irrespective of the install state.

**Comment by Hooverdan96** [comment]
yes, that would be a good approach, too and, I assume, substantially less effort.
